core/domain/playthrough_issue_registry_test.py

```diff
@@ -44,6 +46,17 @@ def tearDown(self) -> None:
         playthrough_issue_registry.Registry._issues = {} # pylint: disable=protected-access
         super().tearDown()
 
+    def test_cannot_get_issues_that_do_not_inherit_base_exploration_issue_spec(
+        self
+    ) -> None:
+        class FakeAction: # pylint: disable=missing-docstring
+            some_property: int
+        swap_getattr = self.swap(
```

Comment:
I don't really understand what the point of this test is (as written) -- given what it's trying to test, wouldn't the right thing to do be to define a new (invalid) class in the module that's being searched through, and ensure that the issue getter doesn't pick it up?